apps/common-app/src/examples/SharedElementTransitions/ImageStack.tsx

```diff
@@ -57,7 +55,7 @@ export function ImageStack({
                 },
                 styles.roundedBorder,
               ]}
-              sharedTransitionTag={`${image}@${index}`}
+              sharedTransitionTag={`SET${index}`}
```

Comment:
`${image}` evaluates to string "[Object Object]", I've tested that the gallery example still works